ams/server/src/main/java/com/netease/arctic/server/RestCatalogService.java

```diff
@@ -106,13 +107,13 @@ public class RestCatalogService extends PersistentBase {
   private final JavalinJackson jsonMapper;
 
   private final TableService tableService;
-  private final MetricsManager metricsManager;
+  private final EventsManager eventsManager;
```

Comment:
EventsManager is a singleton. Do we still need this variable?

Reply:
fixed.